src/app/component/ShowDeliveryRoute/ShowDeliveryRoute.jsx

```diff
@@ -89,11 +90,13 @@ const ShowDeliveryRoute = ({ missions }) => {
   }
 
   return (
-    <Link href={googleMapsUrl} target="_blank">
-      <Button fullWidth={true} variant="text" startIcon={<MuiMapIcon />}>
-        View Route
-      </Button>
-    </Link>
+    <ErrorBoundary>
+      <Link href={googleMapsUrl} target="_blank">
```

Comment:
Was this erroring because of the the external googMapsUrl?

I don't believe `Link` can accept urls that dont point to the site. We should use anchor tag here

Reply:
It wasn't erroring at all but because it uses an external API (the python back end) it seems like an appropriate place for an ErrorBoundary.. if there is something wrong with the python back end it shouldn't take down the app, just the ability to view optimized routes in google maps.

Reply:
fwiw the 
```
 <Link href={googleMapsUrl}
```

Was from master (earlier approved PR). It 'works' like this. Should I still change it to an anchor tag?

Reply:
I think external URLs should be regular anchors right? <a href /> rather than Link--part of the react router api.

Also any time we use target="_blank" We should include a "noopener noreferrer" in a rel attribute to protect from i think tab injection. I forgot the immediate risk that new tabs expose, I'll have to look it up later when i get a minute